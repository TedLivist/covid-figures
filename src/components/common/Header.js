import React from 'react';
import 'boxicons';
import { Link } from 'react-router-dom';

const Header = (props) => {
  const { year, title } = props
  return (
    <div className="d-flex header px-2 py-2">
      <div className="d-flex">
        <Link to="/">
        <box-icon color="white" type='solid' name='chevron-left'></box-icon>
        </Link>
        <div className="text-white">{year}</div>
      </div>
      <div className="text-white">{title}</div>
      <div className="d-flex">
        <div className="me-4"><box-icon color="white" name='microphone'></box-icon></div>
        <div><box-icon color="white" name='cog'></box-icon></div>
      </div>
    </div>
  );
}
 
export default Header;