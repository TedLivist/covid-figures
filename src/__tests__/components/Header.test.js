import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter as Router } from 'react-router-dom';
import Header from '../../components/common/Header';

test('renders correctly', () => {
  const tree = renderer
    .create(
      <Router>
        <Header />
      </Router>
    )
    .toJSON()
  expect(tree).toMatchSnapshot()
})