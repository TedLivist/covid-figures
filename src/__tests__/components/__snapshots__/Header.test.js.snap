// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="d-flex fixed-top header px-2 py-2"
>
  <div
    className="d-flex"
  >
    <a
      href="/"
      onClick={[Function]}
    >
      <box-icon
        color="white"
        name="chevron-left"
        type="solid"
      />
    </a>
    <div
      className="text-white"
    >
      
    </div>
  </div>
  <div
    className="text-white"
  >
    
  </div>
  <div
    className="d-flex"
  >
    <div
      className="me-4"
    >
      <box-icon
        color="white"
        name="microphone"
      />
    </div>
    <div>
      <box-icon
        color="white"
        name="cog"
      />
    </div>
  </div>
</div>
`;
